runtime-management/runtime-management-api/src/main/java/io/cloudslang/runtime/api/python/entities/PythonExecutorDetails.java

```diff
@@ -21,15 +21,17 @@ public class PythonExecutorDetails {
     private String url;
     private String runtimeEncodedAuth;
     private String lifecycleEncodedAuth;
+    private String pythonExecutorPath;
```

Comment:
You are already in the PythonExecutorDetails, no need to call it with a pythonExecutor prefix. Rename to sourceLocation.